src/sprite_character.cpp

```diff
@@ -62,8 +62,7 @@ void Sprite_Character::Update() {
 		int row = character->GetSpriteDirection();
 		auto frame = character->GetAnimFrame();
 		if (frame >= RPG::EventPage::Frame_middle2) frame = RPG::EventPage::Frame_middle;
-		r.Set(frame * chara_width, row * chara_height, chara_width, chara_height);
-		SetSrcRect(r);
+		SetSrcRect({frame * chara_width, row * chara_height, chara_width, chara_height});
```

Comment:
Rect r is now unused in the func -> remove

Reply:
Nice catch, fixed